analyzer/pkg/dpkg/copyright.go

```diff
@@ -0,0 +1,90 @@
+package dpkg
+
+import (
+	"bufio"
+	"bytes"
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/fanal/utils"
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	classifier "github.com/google/licenseclassifier/v2/assets"
+	"io"
+	"regexp"
+	"strings"
+)
+
+const LicenseAdder = "dpkg-license-adder"
+
+var (
+	cl, _                = classifier.DefaultClassifier()
+	copyrightFileRegexp  = regexp.MustCompile(`^usr/share/doc/([0-9A-Za-z_.-]+)/copyright$`)
+	commonLicensesRegexp = regexp.MustCompile(`/?usr/share/common-licenses/([0-9A-Za-z_.+-]+[0-9A-Za-z+])`)
+)
+
+type License struct {
+	Pkg      string
+	Licenses string
+}
+
+// parseCopyrightFile parses /usr/share/doc/*/copyright files
+func parseCopyrightFile(content dio.ReadSeekerAt, filePath string) (*analyzer.AnalysisResult, error) {
+	var licenses []string
+	var buf bytes.Buffer
+
+	tee := io.TeeReader(content, &buf) // Save stream in buffer for re-read with 'licenseclassifier'
```

Comment:
seems `scanner` instance is available within `func (a dpkgAnalyzer) Analyze(...)`

Reply:
Used this scanner, thanks!

Reply:
Used this scanner, thanks!